airlock/business_logic.py

```diff
@@ -77,6 +78,12 @@ def __post_init__(self):
         if not self.root().exists():
             raise BusinessLogicLayer.WorkspaceNotFound(self.name)
 
+    def project(self):
+        return self.metadata.get("project", None)
+
+    def __lt__(self, other):
```

Comment:
I think `@dataclass(order=True)` would mean we don't need this; it compares using the attributes in order, so should still sort properly by name

Reply:
oh, nice will add